examples/simple.js

```diff
@@ -11,12 +10,6 @@ var map = new ol.Map({
       source: new ol.source.OSM()
     })
   ],
-  controls: ol.control.defaults({
-    attributionOptions: /** @type {olx.control.AttributionOptions} */ ({
-      collapsible: false
-    })
-  }),
```

Comment:
From  http://www.openstreetmap.org/copyright/en : For a browsable electronic map, the credit should appear in the corner of the map.

So the collapsed version is sufficient? I simply don't know and faintly remember discussion in the ol3 repository that made us introduce this sort of awkward way to tell the attribution control to not be collapsible.